deptry/cli.py

```diff
@@ -212,6 +213,8 @@ def deptry(
         logging.warning("Missing argument ROOT. E.g. `deptry .`")
         sys.exit(1)
 
+    maybe_install_finder(root)
```

Comment:
I think `maybe` is a bit vague, can you make the function name a bit more explicit? e.g. `install_metadata_finder_if_deptry_not_installed_within_venv`. Or e.g. split them up in an 

```python
if deptry_not_installed_within_venv():
    install_metadata_finder()
```

I think that will make it more clear to anyone reading the code for the first time what is happening here.

Reply:
I liked your second suggestion that played well with grouping arguments in a `ExecutionContext` that makes the eloquent `running_in_project_virtualenv()` more "presentable" at `cli` level.